extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -1,14 +1,32 @@
 <?php
 
-class PortableInfoboxRenderServiceTest extends PHPUnit_Framework_TestCase {
+class PortableInfoboxRenderServiceTest extends WikiaBaseTest {
 	private $infoboxRenderService;
 
 	protected function setUp() {
 		$this->setupFile = dirname( __FILE__ ) . '/../PortableInfobox.setup.php';
 		parent::setUp();
 
-		$mock = $this->getMock( 'PortableInfoboxRenderService', [ 'getThumbnailUrl' ] );
-		$mock->expects( $this->any() )->method( 'getThumbnailUrl' )->will( $this->returnValue( 'http://image.jpg' ) );
+		$mockThumbnailImage = $this->getMockBuilder( 'ThumbnailImage' )
```

Comment:
Yes, I was referring to the test (that's why I changed support => cover in the comment). Thanks!